src/Mutation/UserDelete.php

```diff
@@ -133,12 +133,22 @@ public static function mutate_and_get_payload() {
 			if ( ! function_exists( 'wpmu_delete_user' ) ) {
 				require_once ABSPATH . 'wp-admin/includes/ms.php';
 			}
+			if ( ! function_exists( 'remove_user_from_blog' ) ) {
```

Comment:
Should this be in the is_multisite() check, or am I guilty of premature optimization?

Reply:
nope, I was moving this already! The test actually didn't like it outside the `is_multisite()` call! So good suggestion 😄

Reply:
just pushed this change 😄